src/main/drivers/accgyro_mpu6500.c

```diff
@@ -35,6 +35,25 @@
 #include "accgyro_mpu.h"
 #include "accgyro_mpu6500.h"
 
+static bool mpu6500ReadTemp(int16_t *tempData)
+{
+    uint8_t buf[2];
+    if (!mpuConfiguration.read(MPU_RA_TEMP_OUT_H, 2, buf)) {
+        return false;
+    }
+
+    // RM-MPU-6500A-00.pdf P33
+    // To convert the output of the temperature sensor to degrees C 
+    // please use the formula below:
+    // TEMP_degC = ((TEMP_OUT – RoomTemp_Offset)/Temp_Sensitivity) + 21degC
+    // MPU-6500-Datasheet2.pdf P12
+    // Sensitivity: 333.87
+    // Room Temp Offset: 21C
+    *tempData = 21 + ((int16_t)(buf[0] << 8 | buf[1]) - 21) / 333.87f;
```

Comment:
prefer #defines or constants for the values instead of comments.  especially ones that are used more than once (e.g. `21`)